posthog/api/test/batch_exports/conftest.py

```diff
@@ -115,7 +115,7 @@ def start_test_worker_async(temporal: TemporalClient):
         yield
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture
```

Comment:
Why are we removing the autouse from here? That could lead to some leaking as we clean up temporal schedules as part of this fixture.

Removing the autouse means that tests that do not require this fixture would now be responsible of cleaning up their schedules. Do such cases exist?

Reply:
Another option is to move the `cleanup_temporal_schedules` to a separate, `autouse=True` fixture.

Reply:
Read a bit more, and yeah we now added the `temporal` fixture everywhere. That's okay, but I would separate `cleanup_temporal_schedules` so that `temporal` doesn't have to be required everywhere that doesn't need a temporal client.

Reply:
I just removed it since several of the tests I introduced were just creating models directly in the database and therefore didn't need to start a temporal worker or perform any cleanup. I assumed that temporal schedules would only be created in tests that needed the temporal client, which is why I kept `cleanup_temporal_schedules` in the same fixture. If this is not the case I can move it out into a separate fixture that is always used?

Reply:
I think it's fine. Thank you :+1: